libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
I prefer none of those. Sorry but isn't it obvious that the cluster here means an "rmq" cluster? This cookbook doesn't provision any other clusters AFAIK.